src/components/Chip/Chip.tsx

```diff
@@ -20,16 +20,21 @@ function Chip({ label, selected = false, disabled = false, onClick = () => {} }:
     'cursor-not-allowed': disabled,
     'cursor-pointer': !disabled,
   });
-  
-return (
-    <button
-      type="button"
+
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === "Enter") onClick();
+  };
+
+  return (
+    <div
       className={template}
-      disabled={disabled}
-      onClick={() => onClick()}
```

Comment:
on click is also available for a div

Reply:
@robdembitel yes I am using that, can you please elaborate what you mean?